dll/win32/cryptui/main.c

```diff
@@ -1280,9 +1290,11 @@ static LRESULT CALLBACK cert_mgr_dlg_proc(HWND hwnd, UINT msg, WPARAM wp,
         case IDCANCEL:
             free_certs(GetDlgItem(hwnd, IDC_MGR_CERTS));
             close_stores(GetDlgItem(hwnd, IDC_MGR_STORES));
```

Comment:
It looks strange to do these here , but OK (I don't know yet whether they need to be moved in the WM_DESTROY handler too).